.editorconfig

```diff
@@ -8,5 +8,8 @@ charset = utf-8
 [*.{js,json}]
 indent_size = 2
 
+[web/locales/**.json]
```

Comment:
Good call. At some point we should change that though (in one single PR)

Reply:
I agree, it is strange to have different coding styles for different files.